lib/appsignal/transaction/registry.ex

```diff
@@ -124,6 +116,7 @@ defmodule Appsignal.TransactionRegistry do
   @doc """
   Check if a progress is ignored.
   """
+  @deprecated "Use Appsignal.TransactionRegistry.lookup/1 instead."
```

Comment:
Do we keep a changelog for this kind of changes somewhere? (Not sure if customers are actually using this?)